TMS.Net07.Homework.Calculation/TMS.Net07.Homework.Calculation -easy/Program.cs

```diff
@@ -0,0 +1,158 @@
+    {
+        const decimal valueUSDBYN = 0.3818m, valueEURBYN = 0.315m, valueRUBBYN = 0.0347m;
+        const decimal valueEURUSD = 0.8252m, valueEURRUB = 0.0109m, valueUSDRUB = 0.0132m;
+        const string txtMessage = "Please, enter from what currency exchage(BYN, USD, EUR, RUB)";
+        static void Main(string[] args)
+        {
+            {
+                Console.Title = "Dmitry Salata-TMS.Net07.Homework - easy";
+                decimal exchangeValue, outputExchange;
+                string fromCurrency, toCurrency;
+                bool fromCurrencyIsOk, toCurrencyIsOk, checkExchangeValue;
+
+                Console.WriteLine($"{txtMessage}:");
+                fromCurrency = Console.ReadLine().ToLower();
+                //exit from program
+                for (; !fromCurrency.Equals("exit") ; Console.WriteLine($"{txtMessage} or exit for closing program:"), fromCurrency = Console.ReadLine())
+                {
+                    //check for input. if input is not correct cycle "for" use.
+                    fromCurrencyIsOk = fromCurrency.Equals("usd") || fromCurrency.Equals("eur") || fromCurrency.Equals("rub") || fromCurrency.Equals("byn");
+
+                    for (; fromCurrencyIsOk != true; fromCurrency = Console.ReadLine().ToLower(), fromCurrencyIsOk = fromCurrency.Equals("usd") || fromCurrency.Equals("eur") || fromCurrency.Equals("rub") || fromCurrency.Equals("byn"))
+                    {
+                        Console.WriteLine("Check your input. Enter currency in format BYN, USD, EUR, RUB:");
+                    }
+
+                    Console.WriteLine("Please, enter to what currency exchage (BYN, USD, EUR, RUB):");
+                    toCurrency = Console.ReadLine().ToLower();
+                    toCurrencyIsOk = toCurrency.Equals("usd") || toCurrency.Equals("eur") || toCurrency.Equals("rub") || toCurrency.Equals("byn");
+
+                    for (;toCurrencyIsOk != true; toCurrency = Console.ReadLine().ToLower(), toCurrencyIsOk = toCurrency.Equals("usd") || toCurrency.Equals("eur") || toCurrency.Equals("rub") || toCurrency.Equals("byn")) 
+                    {
+                        Console.WriteLine("Check your input. Enter currency in format BYN, USD, EUR, RUB:");
+                    }
+
+                    Console.WriteLine("Please, enter exchanging amount:");
+                    checkExchangeValue = decimal.TryParse(Console.ReadLine(), out exchangeValue);
+
+                    for (; checkExchangeValue != true; checkExchangeValue = decimal.TryParse(Console.ReadLine(), out exchangeValue)) 
+                    {
+                        Console.WriteLine("Check your enter. Value can be only a number:");
+                    }
+
+                    if (fromCurrency == "usd")
+                    {
+
+                        switch (toCurrency)
+                        {
+                            case ("byn"):
+                                outputExchange = valueUSDBYN * exchangeValue;
```

Comment:
Let's say bank start getting some additional fees for money exchange. In that case, we should add this fee in all lines where we calculating `outputExchange`. It's a good candidate to be moved into a separate method..
Not change request, just advice for the future

Reply:
Ok, I will try to remember this for future work.